tests/manual-test-cases/Group14-Longevity/14-1-Longevity.robot

```diff
@@ -25,15 +25,12 @@ Longevity cleanup
 
 *** Test Cases ***
 Longevity
-    # Each loop should take between 1 and 2 hours
+    # Each regression test takes about 1 minute, therefore each loop should take between 1 to 2 hours
     :FOR  ${idx}  IN RANGE  0  48
-    \   ${rand}=  Evaluate  random.randint(10, 50)  modules=random
+    \   ${rand}=  Evaluate  random.randint(60, 120)  modules=random
+    \   ${date}=  Evaluate  datetime.date.today()  modules=datetime
     \   Log To Console  \nLoop: ${idx}
-    \   Install VIC Appliance To Test Server  debug=%{DEBUG_VCH_LEVEL}  additional-args=%{STATIC_VCH_OPTIONS} %{SYSLOG_VCH_OPTION}
-    \   Repeat Keyword  ${rand} times  Run Regression Tests
-    \   Cleanup VIC Appliance On Test Server
-    \   ${rand}=  Evaluate  random.randint(10, 50)  modules=random
-    \   Install VIC Appliance To Test Server  debug=%{DEBUG_VCH_LEVEL}  certs=${true}  additional-args=%{STATIC_VCH_OPTIONS} %{SYSLOG_VCH_OPTION}
+    \   Install VIC Appliance To Test Server  debug=%{DEBUG_VCH_LEVEL}  additional-args=%{STATIC_VCH_OPTIONS} %{SYSLOG_VCH_OPTION} --container-name-convention longevity-${date}-{id}
```

Comment:
confirmed that `${certs}` defaults to true in `master`.  Not checked 1.4.1 branch.